rebasehelper/tests/test_utils.py

```diff
@@ -150,6 +150,34 @@ def write():
         assert capturer.stdout == 'test stdout'
         assert capturer.stderr == 'test stderr'
 
+    @pytest.mark.parametrize('specification, expected_rgb, expected_bit_width', [
+        ('rgb:0000/0000/0000', (0, 0, 0), 16),
+        ('rgb:ffff/ffff/ffff', (65535, 65535, 65535), 16),
```

Comment:
I don't understand why you still write numeric constants in decimal form instead of hexadecimal.